tree/dataframe/test/dataframe_vary.cxx

```diff
@@ -767,19 +1220,19 @@ TEST_P(RDFVary, VaryTake)
    EXPECT_EQ(sorted(rs["x:1"]), std::vector<int>({1, 2, 3}));
 }
 
-TEST_P(RDFVary, DISABLED_VaryForeach)
+TEST_P(RDFVary, VarySnapshot)
 {
-
-}
-
-TEST_P(RDFVary, DISABLED_VarySnapshot)
-{
-
-}
-
-TEST_P(RDFVary, DISABLED_VaryOnPartialResult)
-{
-
+   const auto fname = "dummy.root";
+   auto h = ROOT::RDataFrame(10)
+               .Define("x", [](ULong64_t e) { return int(e); }, {"rdfentry_"})
+               .Vary(
+                  "x",
+                  [](int x) {
+                     return ROOT::RVecI{x - 1, x + 1};
+                  },
+                  {"x"}, 2)
+               .Snapshot<int>("t", fname, {"x"});
+   EXPECT_THROW(VariationsFor(h), std::logic_error); // not (yet) implemented
```

Comment:
this is ok, I'm actually still not sure how varying a Snapshot should behave :smile: ideas welcome